sos/collector/sosnode.py

```diff
@@ -367,7 +367,8 @@ def _regex_sos_help(self, regex, sosinfo, is_list=False):
             for line in result.splitlines():
                 if not is_list:
                     try:
-                        res.append(line.split()[0])
+                        if line.split():
```

Comment:
We could do something else instead of calling split() twice, let me know if you have any other preference.

Reply:
Yeah I am on torns here. Comparing performance on big text files:

    if line.split():
        print(line.split()[0])

took:

    real	0m5.488s
    user	0m5.090s
    sys	0m0.324s

while:

    ls = line.split()
    if ls:
        print(ls[0])

took:

    real	0m4.161s
    user	0m3.841s
    sys	0m0.252s

Reply:
Then lets go for the second option